bundles/admin/admin-layereditor/view/AdminLayerForm/GeneralTabPane/SrsSettings.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Select, Option, Message, Tooltip, Icon, Tag } from 'oskari-ui';
+import { LocaleConsumer } from 'oskari-ui/util';
+import { StyledComponent } from '../StyledFormComponents';
+import styled from 'styled-components';
+
+const LayerComposingModel = Oskari.clazz.get('Oskari.mapframework.domain.LayerComposingModel');
+
+const FlexRow = styled('div')`
+    display: flex;
+    > * {
+        flex-grow: 1;
+    }
+    > :not(:last-child) {
+        margin-right: 10px;
+    }
+`;
+
+const InfoIcon = styled(Icon)`
+    margin-left: 10px;
+`;
+
+const EpsgCodeTags = ({ codes }) => {
+    if (!codes || !Array.isArray(codes) || codes.length === 0) {
+        return null;
+    }
+    return (
+        <StyledComponent>
+            { codes.map((epsg, i) => <Tag key={i}>{epsg}</Tag>) }
+        </StyledComponent>
+    );
+};
+EpsgCodeTags.propTypes = {
+    codes: PropTypes.any
+};
+
+const SupportedSRS = LocaleConsumer(({ epsgCodes }) => (
+    <div>
+        <Message messageKey='supportedSRS' />
+        <EpsgCodeTags codes={epsgCodes} />
+    </div>
+));
+
+const MissingSRS = LocaleConsumer(({ epsgCodes }) => {
```

Comment:
Does these component need LocaleConsumer-wrapper or is it built in to Message?

Reply:
Yes, that's true, removed LocaleConsumers.